lib/streamlit/hello/util.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018-2019 Streamlit Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+MAX_DOCSTRING = 1000
+
+
+def remove_declaration_and_docstring(lines):
+    """
+    This function parses the source code of a function and removes the function
+    declaration and the docstring if found. If no docstring is found and the
+    function has no code, it returns an empty list. This function can be used
+    in conjunction of inspect.getsourcelines(...) to get the source code
+    lines of a function. If the docstring is longer than MAX_DOCSTRING lines
+    it will not remove the docstring and raise an exception. The function also
+    assumes that the input source code is syntactically correct.
```

Comment:
I wonder we should add a check, like, trying to compile the code. In a way, this is meant to be used by hello.py, but some guarantee against future uses is a good thing. I wonder this is too heavy and the user of this function should guarantee correctness.